tests/browser/playwright.config.ts

```diff
@@ -76,7 +76,12 @@ const smokeTestProjects = [
  */
 const config: PlaywrightTestConfig = {
   testDir: './',
-  testIgnore: ['storybook-docs.test.ts', 'examples.test.ts'],
+  testIgnore: [
+    'storybook-docs.test.ts',
+    'examples.test.ts',
+    '**/__snapshots__/**',
```

Comment:
Can you explain why there's a new snapshots directory? I'm just curious and wonder why we aren't using the one that already exists? I can see these snapshots of a test reported being like a different kind of snapshot from our UI/storybook snapshots.

Reply:
@jack-ryan-nava-pbc, The `__snapshots__` directory contains snapshots generated by unit tests for the custom reporter. It’s separate from the snapshots directory used by browser tests to avoid mixing unrelated test artifacts.